commit b673d12d25d49c36f2f22ca18063d6e63bde8dc6 (refs/changes/66/2666/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-09-27T13:04:16-07:00 (2 years ago)
    
    FWAPI-294 sdc-fwapi 'make docs' broken because fwrule doesn't publish its docs/

diff --git a/.npmignore b/.npmignore
index f0fbb1e..eadfdbb 100644
--- a/.npmignore
+++ b/.npmignore
@@ -3,7 +3,6 @@
 /Makefile
 /coverage
 /deps
-/docs
 /etc
 /src
 /test
diff --git a/package.json b/package.json
index 47bc495..5f2aedb 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "fwrule",
   "description": "Parser for Joyent firewall rules",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "author": "Joyent (joyent.com)",
   "keywords": [ "smartos", "triton" ],
   "repository": {
